packages/side-nav/src/SideNav.story.tsx

```diff
@@ -250,6 +251,7 @@ function MockSideNav() {
           </SideNavGroup>
 
           <SideNavGroup
+            hasActiveItem={hasActiveItem}
```

Comment:
Might just be me, but I'm not seeing anything change when I toggle this on/off

Reply:
Is the nav collapsed? It only affects the collapsed nav.